File: drop_db.js
const { MongoClient } = require('mongodb');

async function run() {
  const uri = "mongodb://localhost:27017";
  const client = new MongoClient(uri);
  try {
    await client.connect();
    const db = client.db('one_system');
    await db.dropDatabase();
    console.log("Database 'one_system' dropped successfully.");
  } finally {
    await client.close();
  }
}

run().catch(console.dir);

File: drop_db.go
package main

import (
	"context"
	"fmt"
	"time"

	mongorepo "one-system-server/internal/infrastructure/persistence/mongodb"
)

func main() {
	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
	defer cancel()

	client, err := mongorepo.NewClient(ctx, "mongodb://localhost:27017")
	if err != nil {
		panic(err)
	}

	err = client.DB("one_system").Drop(ctx)
	if err != nil {
		panic(err)
	}
	fmt.Println("Database dropped successfully.")
}
